doc/refman/RefMan-syn.tex

```diff
@@ -917,6 +917,12 @@ \subsubsection{Binding types of arguments to an interpretation scope}
 codomain of products, and more generally any type argument of a
 declared or defined constant.
 
+\subsection[The {\tt function\_scope} interpretation scope]{The {\tt function\_scope} interpretation scope\index{function\_scope@\texttt{function\_scope}}}
+
+The scope {\tt function\_scope} also has a special status. It is 
+temporarily activated each time a subterm of an expression is expected to be a function.
+IS THERE MORE TO SAY HERE?
```

Comment:
Otherwise, a % in front of the question would maybe prevent accidental merge of the question (though not necessarily ensuring that the question is answered).